classes/warrior.lua

```diff
@@ -1,17 +1,31 @@
 local AddonName, SAO = ...
 
 local function registerClass(self)
+    local tasteforblood = GetSpellInfo(60503);
+    local overpower = GetSpellInfo(7384);
+    local execute = GetSpellInfo(5308);

```

Comment:
I thought `execute` wouldn't work because it was a reserved keyword. Can you confirm it works well?

Reply:
It does work